themes/bootstrap5/templates/ajax/itemCallnumber.phtml

```diff
@@ -0,0 +1,5 @@
+<?php if ($this->callnumberHandler): ?>
+  <a href="<?= $this->url('alphabrowse-home')?>?source=<?=$this->escapeUrl($this->callnumberHandler)?>&from=<?=$this->escapeUrl($this->actualCallnumber)?>"><?=$this->escapeHtml($this->displayCallnumber)?></a>
```

Comment:
I wonder if it would be a little more readable (and slightly more safe) if you built the href URL into a variable in a first step, and then called `escapeHtmlAttr` on it to build the `a` tag.

Also, there's nothing wrong with using `$this->escapeUrl()`, but for whatever reason, we don't use that helper anywhere else in the code, instead calling `urlencode` directly. If you prefer the helper, I don't object, but if it's all the same, maybe it's better to change to `urlencode` for consistency (and if we want to use the helper, we could consider replacing all the `urlencode` calls with helper calls in a separate PR).

Reply:
I changed the `escapeUrl`/`urlencode `to `http_build_query` as i think it has better readability.

Reply:
Thanks, that definitely makes things more readable!